src/components/FieldMappingTab.tsx

```
@@ -0,0 +1,1065 @@
+	const fetchVcFields = async (documentSubType, fieldIndex, docIndex) => {
+		if (!documentSubType) return;
+		try {
+			// Find the document config by subtype
+			const doc = documents.find((d) => d.documentSubType === documentSubType);
+			let vcFields = [];
+			if (doc && doc.vcFieldsRaw) {
+				try {
+					const parsed = JSON.parse(doc.vcFieldsRaw);
+					// Map VC fields to dropdown options
+					vcFields = Object.entries(parsed).map(([key, value]) => {
+						const v: any = value;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)